### ISSUE TITLE
CommonJS export changed from named to default in 1.4.6

### ISSUE THREAD

**Issue by rajsite** [jpillora/xhook#85]
The change in PR #83 changed the export from named to default as described in the following comment: https://github.com/jpillora/xhook/commit/b6cbee57d0dfb24b0cbfc92eeec6a125e9889997#r27245083

Should avoid the breaking API change by changing to a named export:
```coffee
else if typeof module is "object" and module.exports
    module.exports.xhook = xhook
```

**Comment by nowells** [comment]
Or maybe support both (so new feature instead of breaking change)?

```coffeescript
else if typeof module is "object" and module.exports
    module.exports = xhook
    module.exports.xhook = xhook
```

**Comment by morsdyce** [comment]
I'd rather keep it as it was as I don't have enough time to test it thoroughly right now.
I plan to move the codebase to ES6 and rollup which will help doing this in a more sane manner.

I've released 1.4.7 to return to the previous named export

**Comment by nowells** [comment]
Awesome. Thanks for fixing @morsdyce! Sounds like some exciting changes coming. Thanks for writing such a wonderful package.
